.travis.yml

```diff
@@ -10,12 +10,15 @@ rvm:
 - &oldest_ruby 2.3.8
 - jruby-9.2.7.0
 - jruby-9.1.17.0
-# the test suite currently crashes on truffleruby
-#- truffleruby-rc13
 matrix:
   include:
   - rvm: *oldest_ruby
     env: ROUGE_VERSION='~> 2.0.0'
+  - rvm: truffleruby-19.1.0
+    if: env(TRAVIS_EVENT_TYPE) = cron
+    env:
```

Comment:
should we use a single line ?

```yml
# use 3g to avoid OutOfMemoryError when running the tests suite against TruffleRuby
env: TRUFFLERUBYOPT="--vm.Xmx3g"
```

we could even use an inline comment but I'm always a bit scared of how it will get interpreted by the YAML parser:

```yml
env: TRUFFLERUBYOPT="--vm.Xmx3g" # use 3g to avoid OutOfMemoryError when running the tests suite against TruffleRuby
```

Reply:
Either way works for me.